news/1329.md

```diff
@@ -0,0 +1 @@
+When adding a package to (or removing from) a group, enhance the formatting of the group name in the printed message.
```

Comment:
Please rename the fragment file to `1329.feature.md`

Reply:
Will rename as soon as I touch the laptop, thank you!

Reply:
Sorry, I clicked the merge button mistakenly. Never mind I can do it.